app/static/js/vendor/jquery/jquery.codezero.js

```diff
@@ -111,7 +111,7 @@ jQuery.fn.extend({
      */
     setFormLoading: function (isLoading, text) {
         return this.each(function () {
-            if (typeof flag === 'undefined') {
+            if (typeof isLoading === 'undefined') {
```

Comment:
@jack17529 please browse through the issues at https://github.com/fossasia/open-event-orga-server/issues